testing/lib/workarea/storefront/system_test.rb

```diff
@@ -151,6 +151,10 @@ def find_analytics_events(for_event: nil)
           all_events.select { |e| e['name'] == for_event }
         end
       end
+
+      def currency
+        Money.default_currency.symbol
```

Comment:
Wouldn't say I'm "in love" with this approach because I'm not sure how it will jive with multi-site apps that use different currencies per site...

Reply:
Yeah... I'm not loving that either. Did you try or consider refactoring to make the actual symbol irrelevant? Like just test for the amounts. That seems like it would be cleaner in the end. Most if not all these assertions could changed to do that.

More complex ones could be regex like

```
assert(page.has_content?("Next Day - #{currency}20.00"))
# converts to
assert_match(page.text, /Next Day - .20\.00/)
```

or something like that.

Reply:
TIL, `assert_text` takes a regex! :) https://rubydoc.info/github/jnicklas/capybara/master/Capybara/Node/Matchers#has_text%3F-instance_method